gcs/src/main/java/com/google/cloud/hadoop/fs/gcs/GhfsStatistic.java

```diff
@@ -109,6 +110,25 @@ public enum GhfsStatistic {
   STREAM_READ_OPERATIONS(
       StreamStatisticNames.STREAM_READ_OPERATIONS, "Calls of read()", TYPE_DURATION),
 
+  STREAM_RAED_VECTORED_OPERATIONS(
+      StreamStatisticNames.STREAM_READ_VECTORED_OPERATIONS,
+      "Calls of readVectored()",
+      TYPE_DURATION),
+
+  STREAM_READ_VECTORED_READ_BYTES_DISCARDED(
+      StreamStatisticNames.STREAM_READ_VECTORED_READ_BYTES_DISCARDED,
+      "Discarded read bytes during readVectored operation",
+      TYPE_COUNTER),
+
+  STREAM_READ_VECTORED_READ_RANGE_DURATION(
+      "stream_readVectored_range_duration", "Latency of individual FileRange", TYPE_DURATION),
+
+  STREAM_READ_VECTORED_READ_RANGE_SIZE(
+      "stream_readVectored_range_size", "Request content via FileRange i.e. getLength", TYPE_GAUGE),
```

Comment:
If there are multiple FS objects, this metrics will not add much value, correct? What is the purpose of this metric?